CHANGES.md

```diff
@@ -10,6 +10,11 @@
 
 <!-- Changes that affect Black's stable style -->
 
+- Don't move comments along with delimiters, which could cause crashes (#4248)
+
+- Fixed a bug where lines were split incorrectly for function definitions with type
```

Comment:
This needs to be a change to the preview style instead.

Reply:
(And to be clear, the bigger ask here is that the style change itself should only be applied in the preview style, not just that the changelog entry be moved.)

Reply:
Oh, so sorry about that, I misunderstood.